app/src/main/java/net/aiscope/gdd_app/ui/metadata/SampleImagesAdapter.kt

```diff
@@ -86,73 +89,47 @@ private class ImageViewHolder(
         itemView.setOnClickListener { onImageClicked(imageFile, maskFile) }
     }
 
-    fun bind(image: File, mask: File) {
+    fun bind(image: File, mask: File, cacheDir: File) {
```

Comment:
`cacheDir` is not used, did you mean using it below instead of `itemView.context.cacheDir`? Otherwise, it can be removed.